src/App.scss

```diff
@@ -0,0 +1,153 @@
+// variables
+$colorGreen: #cdc6ac;
+$lightGreen: #e0e1d8;
+$colorBrown: #a85d36;
+$colorPurple: #b0a19d;
+$quizFont: "IM Fell DW Pica", serif;
+
+body {
+  background-color: $colorGreen;
+  font-family: $quizFont;
+  margin: 0 auto;
+}
+
+h1 {
+  font-size: 62px;
+  font-style: italic;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+h2 {
+  font-size: 54px;
+  font-style: italic;
+  text-decoration: overline;
+  margin: 0;
+}
+
+.tv-header {
+  background-image: url(./assets/pattern_overlapping-hexagons_0.5_2_0-0_0_1__cdc6ac_ffffff.png);
+  //   background-repeat: no-repeat;
+  //   background-size: 100%;
+  .header-container {
+    display: flex;
+    justify-content: center;
+    .header-text {
+      width: 50%;
+      color: $colorBrown;
+      line-height: 5rem;
+      padding-top: 200px;
+      padding-left: 80px;
+    }
+  }
+  .text-container {
+    width: 50%;
+  }
+  .image-container {
+    width: 50%;
+    .header-image {
+      background-color: transparent;
+      max-width: 550px;
+      position: relative;
+      top: 6px;
+      left: 10px;
+    }
+  }
+}
+
+#page-container {
```

Comment:
They might deduct marks for using an ID tag # for styling, you can add a class name to the element, in addition to an ID